index.css

```diff
@@ -0,0 +1,79 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins:wght@200;300;500;700&display=swap");
+
+body {
+  background: #f7f7f9;
+  margin: 0;
+  font-family: "Poppins", sans-serif;
+}
+
+.nav-bar {
+  background-color: #f7f7f9;
+  padding: 20px 15px;
+}
+
+.nav-list {
+  display: flex;
+  justify-content: space-between;
+}
+
+.logo {
+  font-family: "Poppins", sans-serif;
+  font-style: normal;
+  font-weight: bold;
+  line-height: 20px;
+  font-size: 18px;
+  text-decoration: none;
+  color: #6070ff;
+}
+
+.menu-button {
+  border: none;
+  background-color: #f7f7f9;
+}
+
+.container {
+  background-image: url("./static/images/header-shapes\ mobile.png");
+  background-repeat: no-repeat, no-repeat;
+  background-size: cover, 100% 100%;
+  padding: 114px 24px;
+}
```

Comment:
I think there are some little differences between your work and the Figma design😉

this is the design
<img width="370" alt="design" src="https://user-images.githubusercontent.com/64643995/212049298-ff8ec1c9-99e0-4411-ab03-6b0bb1ff8a1c.png">

this is yours
<img width="262" alt="yours" src="https://user-images.githubusercontent.com/64643995/212049406-0ed9d76b-3f91-4dab-8389-b6af830be720.png">

According to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1&t=1BxFWDtrqqjloCcy-0), there is a curve on the left bottom corner. also, the section has a background color different from the body, to make that curve visible. Kindly implement that feature🤗